src/harness/testcases/WINNF_FT_S_PCR_testcase.py

```diff
@@ -92,34 +60,79 @@ def isPpaWithinServiceArea(pal_records, ppa_zone_geometry):
   return ppa_zone_shapely_geometry.buffer(-1e-6).within(pal_service_area)
 
 
-class PpaCreationTestcase(sas_testcase.SasTestCase):
-  """Implementation of PCR tests to verify the area of the non-overlapping
-  difference between the maximum PPA boundary created by SAS UUT shall be no more than
-  10% of the area of the maximum PPA boundary created by the Reference Model.
+def assertRegConditionalsForPpaRefModel(registration_requests,
+                                        conditional_registration_datas):
+  """Check the REG Conditionals for PPA creation model and raises an exception.
+
+  Performs the assert to check installationParam present in
+  registrationRequests or conditional registration data and raises an exception.
+  PpaCreationModel requires the input registrationRequests to have
+  'installationParam'. But this parameter is removed for devices where
+  conditionals are pre-loaded. Adding the 'installationParam' into
+  registrationRequests by taking the corresponding values from
+  conditionalRegistrationData.
+
+  Args:
+    registration_requests: A list of individual CBSD registration
+      requests (each of which is itself a dictionary).
+    conditional_registration_datas: A list of individual CBSD registration
```

Comment:
`conditional_registration_data` -- 'data' is already plural.

Reply:
changed all occurrences of "conditional_registration_datas" to "conditional_registration_data"